Public/Sdk/Public/Managed/Tools/Csc/csc.dsc

```diff
@@ -266,6 +268,11 @@ export interface Arguments extends Transformer.RunnerArguments{
     defines?: string[];
     /** This option instructs the compiler to only use ISO-1 C# language features, i.e which basically boils down to C# 1.0 language features. */
     languageVersion?: string;
+    
+    /** Specify nullable context option enable|disable. */
+    nullable?: boolean;
+    /** Specify nullable context option enable|disable|safeonly|warnings|safeonlywarnings.*/
+    nullabilityContext?: NullabilityContext;

```

Comment:
Why do you need both `nullable` and `nullabilityContext`?  It looks like `nullabilityContext` subsumes `nullable`.

Reply:
I think 'nullable: bool' is just a shorthand for nullabilityContext: NullabilityContext.enabled.

I decided to expose the same options that csc exposes.